packages/mobile/android/gradle.properties

```diff
@@ -22,8 +22,8 @@ CELO_RELEASE_STORE_FILE=celo-release-key.keystore
 CELO_RELEASE_KEY_ALIAS=celo-key-alias
 
 # Setting this manually based on version number until we have this deploying via Cloud Build
-# Example: v1.5.1 deployment number 1 = 100500101 (1 005 001 01)
-VERSION_CODE=100800202
+# Example: v1.5.1 deployment number 1 = 1005001001 (1 005 001 001)
+VERSION_CODE=1000100001
```

Comment:
Should this be 1000001001 According to the new scheme here?

Reply:
This seems like the right number of zeroes to me: 1 000 100 001 - is this different from how you interpreted the scheme above?

Reply:
Actually looking at this again - yes it should be but since the build has been deployed we can't bump it down. Fortunately the next build will be `1 100 000 001` and this will be resolved